pyfr/integrators/base.py

```diff
@@ -33,12 +30,7 @@ def __init__(self, backend, systemcls, rallocs, mesh, initsoln, cfg):
             stats = Inifile(initsoln['stats'])
             self.tcurr = stats.getfloat('solver-time-integrator', 'tcurr')
 
-            # Cull already written output times
-            self.tout = [t for t in self.tout if t > self.tcurr]
-
-        # Ensure no time steps are in the past
-        if self.tout[0] < self.tcurr:
-            raise ValueError('Output times must be in the future')
+        self.tlist = np.array([self.tend])
```

Comment:
If you need a list of values and you intend to extend the list then the most sensible type is a `list`.  Moreover, NumPy arrays can not be extended, lists can.